Islandora-CLAW/islandora_defaults
# Add RDF mapping of field_weight as co:index (Issue 1262)

**GitHub Issue**: Islandora-CLAW/CLAW/issues/1262

Replaces #6  

# What does this Pull Request do?

~~Creates a "Reorder Children" view/tab using the integer_weight_selector provided by the accompanying PR.~~

Adds an RDF mapping of field_weight as co:index. ('cause everything else in this PR is moving to core: Islandora-CLAW/islandora/pull/171)

# What's new?

* Added the RDF mapping for field_weight
* Does this change require documentation to be updated? We should add some documentation.
* Does this change add any new dependencies? No.
* Does this change require any other modifications to be made to the repository
 (ie. Regeneration activity, etc.)? Re-index for weight values to appear.
* Could this change impact execution of existing code? No.

# How should this be tested?

- Apply the PR
- Clear cache
- View the jsonld of a node with a field_weight value and see the co:index predicate with the appropriate value.

# Additional Notes:

Although we talked about replacing the 'purl.org/co/index' with 'schema:position', integers are serialized with the type `http://www.w3.org/2001/XMLSchema#int`; which Google's structured data tester did not like. So, at least for now, we should keep the 'co:index' property until we decide if trying to rewrite Drupal integer types as 'schema:Integer'.

# Interested parties
@dannylamb, @Islandora-CLAW/committers
